examples/next/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@aws-amplify/geo": "^3.0.29",
+    "@aws-amplify/geo": "3.0.31",
```

Comment:
Intentional to lose the range?

Reply:
Yeah, just to align with how the other example apps declare deps